### PR TITLE
Feature/Hero

### PR DESCRIPTION
**Pull request by gustaveen**
New Hero component

Screenshot 2020-05-19 at 10 46 45

### CODE REVIEW

File: src/blocks/HeroBlock/HeroBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+  ${TABLET_BP_UP} {
+    margin-bottom: 1.25rem;
+  }
+
+  br {
+    display: none;
+    ${MOBILE_BP_UP} {
+      display: block;
+    }
+  }
```
Comment (gustaveen) [inline src/blocks/HeroBlock/HeroBlock.tsx:93]: In order to have control over linebreaks in desktop and to let it flow in mobile

File: src/components/blockHelpers.tsx [hunk 2]
```diff
@@ -250,16 +250,31 @@ export const getMinimalColorStyles: ColorSetGetter<minimalColorComponentColors>
...
+  return {
+    backgroundImage: `${bgTint}url(${backgroundImage})`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center center',
+    backgroundRepeat: 'no-repeat',
+
+    [TABLET_BP_DOWN]: {
+      backgroundImage: `${bgTint}url(${
+        backgroundImageMobile === '' ? backgroundImage : backgroundImageMobile
+      })`,
```
Comment (gustaveen) [inline src/components/blockHelpers.tsx:275]: Might not be the most beautiful solution, would like to omit this css completely if there is no `backgroundImageMobile`

Comment (palmenhq) [reply src/components/blockHelpers.tsx:275]: what about something like

```ts
[TABLET_BP_DOWN]: backgroundImageMobile ? {
  backgroundImage: ...
} : {},
```

**Review by palmenhq — COMMENTED** [review]
amaze!

File: src/components/blockHelpers.tsx (hunk 2 above)
Comment (gustaveen) [reply src/components/blockHelpers.tsx]: Ah yes, thank you! 🙏

File: src/blocks/HeroBlock/HeroBlock.tsx (hunk 1 above)
Comment (gustaveen) [reply src/blocks/HeroBlock/HeroBlock.tsx:93]: hehe thank you, so simple but but solves the problem 😄

File: src/components/blockHelpers.tsx [hunk 3]
```diff
@@ -250,16 +250,31 @@ export const getMinimalColorStyles: ColorSetGetter<minimalColorComponentColors>
...
+    : ''
+
+  return {
+    backgroundImage: `${bgTint}url(${backgroundImage})`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center center',
+    backgroundRepeat: 'no-repeat',
+
+    [TABLET_BP_DOWN]: {
+      backgroundImage: `${bgTint}url(${
```
[the remark this answers is not in the document]
Comment (gustaveen) [reply src/components/blockHelpers.tsx]: True that!

File: src/components/blockHelpers.tsx (hunk 2 above)
Comment (gustaveen) [reply src/components/blockHelpers.tsx]: Updated: https://github.com/HedvigInsurance/web-next/pull/458/commits/0de1df27ff13f31bf691d9653c433c163fd897eb

**Review by palmenhq — DISMISSED** [review]
Boom! Nice

**Pull request merged**